### PR TITLE
Remove previous public README

### PR DESCRIPTION
**Pull request by edmulraney**
Our `liquity/liquity` and `liquity/dev` README diverged. 

Since `liquity/liquity` is archived and out of date, we can delete the README.public.md file which was used by `git-filter-repo` to produce the `liquity/liquity` README.md.

I've merged the content into our current README so nothing is lost.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1409,6 +1456,110 @@ yarn rebuild
 
 This combines the top-level `prepare` and `build` scripts.
 
+You'll find the output in `packages/dev-frontend/build`.
+
+### Configuring your custom Dev UI
```
Comment (cvalkan) [inline README.md:1461]: Perhaps we should replace "Dev UI" by "Frontend" throughout the Readme.

**Review by cvalkan — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (edmulraney) [reply README.md]: Good idea - done: https://github.com/liquity/dev/pull/529/commits/a88ac4964ed4a03beb43f4d7afdf007de0eb16b1

File: README.md [hunk 2]
```diff
@@ -1,107 +1,154 @@
...
+than by active governance or monetary interventions.
+
+The protocol has built-in incentives that encourage both early adoption and the operation of
+multiple front ends, enhancing decentralization.
+
+## More information
+
+Visit [liquity.org](https://www.liquity.org) to find out more and join the discussion.
+
+### Packages
```
Comment (danielattilasimon) [inline README.md:25]: This section is a bit of a duplicate of https://github.com/liquity/dev#project-structure

**Review by danielattilasimon — APPROVED** [review]
Looks good! 👍 

Left 1 minor comment.

File: README.md (hunk 2 above)
Comment (edmulraney) [reply README.md]: Good spot - fixed: https://github.com/liquity/dev/pull/529/commits/40f7f5f64fbda7eaa2b2141e4902358869ecb194

**Pull request merged**